x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/analysis/analyzer/Verifier.java

```diff
@@ -189,11 +189,12 @@ public Verifier(Metrics metrics) {
             plan.forEachExpressionsUp(e -> {
                 if (e instanceof Alias) {
                     Alias a = (Alias) e;
-                    collectRefs.put(a.toAttribute(), a.child());
+                    Attribute attr = a.toAttribute();
+                    collectRefs.put(attr, a.child());
```

Comment:
Will merge these two lines.

Reply:
You need the checks to pass before merging so you'll have to do it anyway, better do it sooner rather than later.